miyagi_trainer/train.py

```diff
@@ -263,6 +264,9 @@ def train(args):
     parser.add_argument("--train_datasets", action='store', type=str, nargs="+", required=True)
     parser.add_argument("--val_datasets", action='store', type=str, nargs="+", required=True)
     parser.add_argument("--balanced_weights", action=argparse.BooleanOptionalAction)
+    parser.add_argument("--multiple_datasets_temperature", type=float, required=False,
```

Comment:
Didn't get that, the temperature is not suppose to be defined by a value for each dataset? Is there any reference for this type of weighting?

It would be awesome if you explain a bit this options in the README.md as well.

Reply:
It was a hacky way to balance multiple datasets that was not very good, honestly. I think it is a nice feature to have, but I would reimplement it with a more clear interface and output. Probably something like passing a list of datasets: [ds1, ds2, ds3] and some sampling weights to them: [0.2, 0.1, 0.7] that would sample 20% of ds1, 10% of ds2 and 70% of ds3. Sounds better, right?